src/core/resolvers/block.ts

```diff
@@ -36,6 +37,22 @@ function transactionFilter({ filter }: TransactionFilter): (tx: EthqlTransaction
   return tx => withInput(tx) && contractCreation(tx) && withLogs(tx);
 }
 
+/**
+ * Gets the ommer block.
+ */
+async function ommers(
+  obj: EthqlBlock,
+  _,
+  { ethService }: EthqlContext,
+  info: GraphQLResolveInfo,
+): Promise<EthqlOmmerBlock[]> {
+  let ommers: EthqlOmmerBlock[] = [];
+  for (const index of obj.uncles.keys()) {
+    ommers.push(await ethService.fetchOmmerBlock(obj.hash, index));
```

Comment:
Several comments here:

* There's opportunity to parallelise the fetching of all ommer blocks. This logic fetches them serially.
* Why this iteration logic? There are simpler ways to iterate over elements of an array ;-)